presto-iceberg/src/main/java/com/facebook/presto/iceberg/changelog/ChangelogSplitSource.java

```diff
@@ -78,7 +78,8 @@ public ChangelogSplitSource(
             ConnectorSession session,
             TypeManager typeManager,
             Table table,
-            IncrementalChangelogScan tableScan)
+            IncrementalChangelogScan tableScan,
+            long snapshotSequenceNumber)
```

Comment:
```suggestion
            IncrementalChangelogScan tableScan)
```

nit: missed cleaning this?